### ISSUE TITLE
Bug: Blocking Feature not working

### ISSUE THREAD

**Issue by anurag0006** [PalisadoesFoundation/talawa-admin#732]
**Describe the bug**
The blocking feature for the users who spammed is not working
**To Reproduce**

1.Login into talawa-admin
2.Click on manage button for any organization
3.Now click on block/unblock button from navbar
4. Now try to block any user.

**Expected behavior**
The block functionality should work properly
**Actual behavior**
the feature isn't working

**Screenshots**
![image](https://user-images.githubusercontent.com/107215525/225734573-8e893379-22be-4084-9825-804391aeb403.png)

**Comment by anurag0006** [comment]
I am made the PR for my existing Issues and free to work so would like to work on this

**Comment by adi-uchiha** [comment]
@anurag0006 It is because the person you are trying to block is not the part of the org. The list renders the entire database of users which a another bug and is worked on by someone I guess

**Comment by nishtha981** [comment]
can i be assigned this issue?

**Comment by palisadoes** [comment]
Unassigning due to inactivity and no submitted PR. Others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue.

**Comment by VictoriaAde** [comment]
Assign this issue to me, please.

**Comment by anurag0006** [comment]
Yeah sorry i didn't worked on the issue, I forgot to close this issue as the Problem for this on the API side, we need to mutate the queries on the backend to make this feature work
